src/Pages/Wallets.razor

```diff
@@ -112,26 +114,26 @@
-                <DataGridColumn TItem="Wallet" Displayable="false" Filterable="true" Caption="Compromised" Field="@nameof(Wallet.IsCompromised)" Editable="true" CellsEditableOnNewCommand="false" CustomFilter="((searchItem, searchValue) => OnCustomFilter(searchItem, _compromisedWalletFilter))"/>
+                <DataGridColumn TItem="Wallet" Displayable="false" Filterable="true" Caption="Cold Wallet" Field="@nameof(Wallet.IsHotWallet)" Editable="false"/>
+                <DataGridColumn TItem="Wallet" Displayable="false" Filterable="true" Caption="Finalised" Field="@nameof(Wallet.IsFinalised)" Editable="false"/>
+                <DataGridColumn TItem="Wallet" Displayable="false" Filterable="true" Caption="Archived" Field="@nameof(Wallet.IsArchived)" Editable="true" CellsEditableOnNewCommand="false"/>
+                <DataGridColumn TItem="Wallet" Displayable="false" Filterable="true" Caption="Compromised" Field="@nameof(Wallet.IsCompromised)" Editable="true" CellsEditableOnNewCommand="false"/>
                 <DataGridSelectColumn TItem="Wallet" Sortable="false" Filterable="true"  Displayable="@IsColumnVisible(WalletsColumnName.WalletProperties)">
                     <FilterTemplate>
                         <Dropdown>
                             <DropdownToggle>Select</DropdownToggle>
                             <DropdownMenu>
                                 <div class="checkbox-drop">
-                                    <Check TValue="bool" @bind-Checked="_hotWalletFilter">🔥 Hot Wallet</Check>
-                                    <Check TValue="bool" @bind-Checked="_coldWalletFilter">❄️ Cold Wallet</Check>
-                                    <Check TValue="bool" @bind-Checked="_finalizedWalletFilter">🏁 Finalized</Check>
-                                    <Check TValue="bool" @bind-Checked="_archivedWalletFilter">📂 Archived</Check>
-                                    <Check TValue="bool" @bind-Checked="_compromisedWalletFilter">⛔️ Compromised</Check>
+                                    <Check TValue="bool"  CheckedChanged="@(() => { _hotWalletFilter = !_hotWalletFilter; OnCheckedChanged(true); })">🔥 Hot Wallet</Check>
+                                    <Check TValue="bool"  CheckedChanged="@(() => { _coldWalletFilter = !_coldWalletFilter; OnCheckedChanged(true); })">❄️ Cold Wallet</Check>
+                                    <Check TValue="bool"  CheckedChanged="@(() => { _finalisedWalletFilter = !_finalisedWalletFilter; OnCheckedChanged(true); })">🏁 finalised</Check>
+                                    <Check TValue="bool"  CheckedChanged="@(() => { _archivedWalletFilter = !_archivedWalletFilter; OnCheckedChanged(true); })">📂 Archived</Check>
+                                    <Check TValue="bool"  CheckedChanged="@(() => { _compromisedWalletFilter = !_compromisedWalletFilter; OnCheckedChanged(true); })">⛔️ Compromised</Check>
```

Comment:
You could create a function for not repeating so much code. And I don't know how dotnet handles this, but most probably every time the component rerenders, new lambda functions are created for every Check tag